dyno-core/src/main/java/com/netflix/dyno/connectionpool/Host.java

```diff
@@ -171,6 +199,10 @@ public int compareTo(Host o) {
 
     @Override
     public String toString() {
+        if (this.hashtag  != null){
```

Comment:
Try combining the two returns. You do not really require a if else here.

Reply:
Would this be better

```
    @Override
    public String toString() {
        
        String host ="Host with Hashtag [hostname=" + hostname + ", ipAddress=" + ipAddress + ", port=" + port + ", rack: " + rack
                + ", datacenter: " + datacenter + ", status: " + status.name();
        if (this.hashtag  != null){
            return host + ", hashtag=" + hashtag + "]";
        }
        return host + "]";
    }
```

Reply:
String host ="Host [hostname=" + hostname + ", ipAddress=" + ipAddress + ", port=" + port + ", rack: " + rack
                + ", datacenter: " + datacenter + ", status: " + status.name() + ", hashtag=" + hashtag + "]";

Test it!

Reply:
Done!